contractcourt/contract_resolvers.go

```diff
@@ -169,45 +169,17 @@ func (h *htlcTimeoutResolver) Resolve() (ContractResolver, error) {
 	// spent, and the spending transaction has been fully confirmed.
 	waitForOutputResolution := func() error {
 		// We first need to register to see when the HTLC output itself
-		// has been spent so we can wait for the spending transaction
-		// to confirm.
+		// has been spent by a confirmed transaction.
 		spendNtfn, err := h.Notifier.RegisterSpendNtfn(
 			&h.htlcResolution.ClaimOutpoint,
-			h.broadcastHeight, true,
+			h.broadcastHeight, false,
 		)
 		if err != nil {
 			return err
 		}
 
-		var spendDetail *chainntnfs.SpendDetail
 		select {
-		case s, ok := <-spendNtfn.Spend:
-			if !ok {
-				return fmt.Errorf("notifier quit")
-			}
-
-			spendDetail = s
-
-		case <-h.Quit:
-			return fmt.Errorf("quitting")
-		}
-
-		// Now that the output has been spent, we'll also wait for the
```

Comment:
We still may want to actually wait for an additional, or set of additional confirmations...

Reply:
Note that this is equivalent to the previous logic:

Before this commit we wait for a spend (any spend, even mempool), then check that it has at least 1 conf.
After this commit we wait for a confirmed spend.

Are you suggestion that we should wait for additional confirmations of the spend? Should we do 6 conf?

Reply:
Yes suggesting that we may want to wait for additional confirmations. This isn't a blocker for this PR though, as it would be a project wide change if we went down this route.